joint_state_controller:
  type: joint_state_controller/JointStateController
  publish_rate: 500

effort_group_controller:
  type: effort_controllers/JointGroupEffortController
  joints: [joint_0, joint_1, joint_2, joint_3, joint_4, joint_5, joint_6, joint_7, joint_8, joint_9,
           joint_10, joint_11, joint_12, joint_13, joint_14, joint_15, joint_16, joint_17, joint_18, joint_19,
           joint_20, joint_21, joint_22, joint_23, joint_24, joint_25, joint_26, joint_27, joint_28, joint_29]
